src/UniOneClient.php

```diff
@@ -11,110 +11,134 @@
 /**
  * This class is the base class for the UniOne SDK.
  */
-final class UniOneClient {
+final class UniOneClient
+{
+    /**
+     * The HTTP client.
+     *
+     * @var Client
+     */
+    private Client $httpClient;
 
-  /**
-   * The HTTP client.
-   *
-   * @var Client
-   */
-  protected Client $httpClient;
+    /**
+     * UniOne Instance.
+     *
+     * @var string|UriInterface
+     * @todo: Make it configurable.
+     */
+    private static $endpoint = 'https://eu1.unione.io/en/transactional/api/v1/';
 
-  /**
-   * UniOne Instance.
-   *
-   * @var string|UriInterface
-   * @todo: Make it configurable.
-   */
-  static protected $endpoint = 'https://eu1.unione.io/en/transactional/api/v1/';
+    /**
+     * The API key.
+     *
+     * @var string
+     */
+    private string $apiKey;
 
-  /**
-   * The API key.
-   *
-   * @var string
-   */
-  protected string $apiKey;
+    /**
+     * The timeout in seconds.
+     *
+     * @var float
+     */
+    private float $timeout;
 
-  public function __construct() {
-    $this->httpClient = new Client([
-      'base_uri' => $this::$endpoint
-    ]);
-  }
+    public function __construct()
+    {
+        $this->timeout = 5;
+        $this->httpClient = new Client([
+          'base_uri' => $this::$endpoint,
```

Comment:
We can pass a timeout into Guzzle client constructor and it will be applicable to all requests unless other is specified. I think it would be nice to expose the HTTP client so developers can set anything they want (not just the timeout). Don't worry about it now, we'll do it in a follow-up task.